SpiffWorkflow/bpmn/PythonScriptEngine.py

```diff
@@ -26,66 +27,6 @@
 # 02110-1301  USA
 
 
-class Box(dict):
```

Comment:
I like giving people the option of not using Box.

Reply:
I think we should deprecate box too.  The ultimate goal here should be to provide a default that does nothing but execute and evaluate with the task data added to the context, with no additional changes to anything outside of execution/evaluation.

Reply:
Sounds good.

Reply:
Added a deprecation warning and a `BoxedTaskDataEnvironment` if consumers need/want it. Updated two tests that relied on Box to use this new environment.